app/models/user.rb

```diff
@@ -153,13 +153,14 @@ def active_mhv_ids
   end
 
   def address
-    address = mpi_profile&.address
+    address = identity&.address || mpi_profile&.address
     {
-      street: address&.street,
-      city: address&.city,
-      state: address&.state,
-      country: address&.country,
-      zip: address&.postal_code
+      street: address[:street],
+      street2: address.instance_of?(Hash) ? address[:street2] : nil,
```

Comment:
I don't think we should use object introspection to determine the behavior of this field, it's not a very clear way to differentiate the fact that the `identity` version of this object has the field and the `mpi` version doesn't. 

You could add a more clear conditional in this function, or, one idea I had, we could just add `street2` to `mvi_profile_address.rb`. We won't currently write to it but it'd be fine just always being `nil` for now and maybe in the future we could investigate if there are multiline streets in MPI